haskell-overlays/reflex-packages.nix

```diff
@@ -90,8 +90,8 @@ in
   monoidal-containers = self.callCabal2nix "monoidal-containers" (fetchFromGitHub {
     owner = "obsidiansystems";
     repo = "monoidal-containers";
-    rev = "79c25ac6bb469bfa92f8fd226684617b6753e955";
-    sha256 = "0j2mwf5zhz7cmn01x9v51w8vpx16hrl9x9rcx8fggf21slva8lf8";
+    rev = "11eec0a1da953fe23ee494f9a66235e48c55a17f";
```

Comment:
this should instead be a clean version from hackage or if it's not in all-cabal-hashes it should come from the upstream repo and not obsidian's fork